appspec-scripts/start_app.sh

```diff
@@ -1,10 +1,25 @@
 #!/bin/bash
-set -e
-export PATH=$PATH:/home/ubuntu/.cargo/bin
+set -x
```

Comment:
we dont need -e for early termination anymore?

Reply:
**set -e** will make the script to exit immediately, if any command returns a non-zero exit code.
The way this script is implemented, defines a list of ports and loops through each one of them; then, it executes netstat to check if the port is in use. If not, **netstart.+egrep** command returns RC=1, causing the script to exit. This is why it was decided to eliminate this specific parameter.

Reply:
i see, i'm just worried that an uncaught error from a command in these scripts eventually causes the service to be deployed wrongly; but since we have the validate_app check, it seems like it should be fine?

Reply:
> i see, i'm just worried that an uncaught error from a command in these scripts eventually causes the service to be deployed wrongly; but since we have the validate_app check, it seems like it should be fine?

Correct! :)